src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -5092,6 +5090,21 @@ namespace LowTempRadiantSystem {
         }
     }
 
+    Real64 RadiantSystemBaseData::calculateOperationalFraction(Real64 const offTemperature, Real64 const controlTemperature, Real64 const throttlingRange)
```

Comment:
Hooray for a base class method!

Reply:
Let's just say I had a great teacher!  I was definitely following examples from the previous radiant work.  Glad to be doing things in a way that heads E+ in the right direction.